lib/temp-directory.js

```diff
@@ -1,13 +1,10 @@
-import { promisify } from 'util';
 import { join } from 'path';
 import { promises as fs } from 'fs';
 
-import rimrafLib from 'rimraf';
+import rimraf from 'rimraf';
```

Comment:
Consider switching to `fs.promises.rm` (available as of v14.14.0)

Reply:
https://github.com/nodejs/citgm/pull/940